pyproject.toml

```
@@ -11,7 +11,15 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
```

The installation of the `cothread` dependency fails on python 3.11. Also we use 3.10 in the containers so should be consistent.

```suggestion
    "Programming Language :: Python :: 3.10",
```